gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/resources/organizations/ReportersResource.java

```diff
@@ -0,0 +1,137 @@
+
+import io.gravitee.am.identityprovider.api.User;
+import io.gravitee.am.management.handlers.management.api.resources.AbstractResource;
+import io.gravitee.am.management.service.ReporterPluginService;
+import io.gravitee.am.management.service.ReporterServiceProxy;
+import io.gravitee.am.model.Acl;
+import io.gravitee.am.model.Reference;
+import io.gravitee.am.model.ReferenceType;
+import io.gravitee.am.model.Reporter;
+import io.gravitee.am.model.permissions.Permission;
+import io.gravitee.am.service.OrganizationService;
+import io.gravitee.am.service.exception.OrganizationNotFoundException;
+import io.gravitee.am.service.model.NewReporter;
+import io.gravitee.common.http.MediaType;
+import io.reactivex.rxjava3.core.Single;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.ArraySchema;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.inject.Inject;
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.PathParam;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.container.AsyncResponse;
+import jakarta.ws.rs.container.ResourceContext;
+import jakarta.ws.rs.container.Suspended;
+import jakarta.ws.rs.core.Context;
+import jakarta.ws.rs.core.Response;
+
+import java.net.URI;
+
+@Tag(name = "reporter")
+public class ReportersResource extends AbstractResource {
+    @Context
+    private ResourceContext resourceContext;
+    private final ReporterServiceProxy reporterService;
+    private final ReporterPluginService reporterPluginService;
+    private final OrganizationService organizationService;
+
+    @Inject
+    public ReportersResource(ReporterServiceProxy reporterService,
+                             ReporterPluginService reporterPluginService,
+                             OrganizationService organizationService) {
+        this.reporterService = reporterService;
+        this.reporterPluginService = reporterPluginService;
+        this.organizationService = organizationService;
+    }
+
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(summary = "List registered reporters for a security domain",
+            description = "User must have the DOMAIN_REPORTER[LIST] permission on the specified domain " +
+                    "or DOMAIN_REPORTER[LIST] permission on the specified environment " +
+                    "or DOMAIN_REPORTER[LIST] permission on the specified organization. " +
+                    "Except if user has DOMAIN_REPORTER[READ] permission on the domain, environment or organization, each returned reporter is filtered and contains only basic information such as id and name and type.")
```

Comment:
TODO: permission shouldn't be related to the domain or env here as we are at Org level, this is OK into the code but this description needs to be changed